src/package-linker.js

```diff
@@ -251,10 +251,41 @@ export default class PackageLinker {
       }
     }
 
-    // linked modules
-    for (const loc of possibleExtraneous) {
-      const stat = await fs.lstat(loc);
+    // if an extraneous is an entry created via "yarn link", we prevent it from being overwritten
```

Comment:
Nope, in this context `extraneous` is used as a name (like in `possibleExtraneous`)